readded everything (#104)
MrLongbottom/P10-Article: sections/futureWork.tex

> Due to the promising results provided by the modified pachinko model, investigating this model further might be beneficial to incorporating metadata into topic models.

Reviewer: ```suggestion
Due to the promising results provided by the modified \gls{pam}, investigating this model further might be beneficial to incorporating metadata into topic models.
```

However, testing these models on multiple datasets needs to be accounted for since the generalizability of these models is not explored within this paper.
Using word embedding to further improve the performance of models can also be viewed as a possible next step for this project since a wide number of papers are using this technique to improve topic modeling\cite{MetaLDA2017}\cite{dieng2020topic}.

> It would also be interesting to see how we could incorporate these models into an existing IT infrastructure at Nordjyske.

Reviewer: ```suggestion
It would also be interesting to see how we could incorporate these models into an existing IT infrastructure for a news site such as Nordjyske.
```

The next step in that process would be to investigate which part of their infrastructure could benefit from the use of topic modeling, whether it is for recommendation or automatic tagging of articles. 

> We have written about a few potential use cases of our project in Section \ref{sec:appendix_applications} in the Appendix. 

Reviewer 2: ```suggestion
We have written about a few more potential use cases of our project in Section \ref{sec:appendix_applications} in the Appendix. 
```

> We have written about a few potential use cases of our project in Section \ref{sec:appendix_applications} in the Appendix. 

Reviewer: ```suggestion
We have written about a few potential use cases of our project in Appendix \autoref{sec:appendix_applications}. 
```

Reviewer 2: ```suggestion
We have written about a few more potential use cases of our project in Appendix \autoref{sec:appendix_applications}. 
```
